### PR TITLE
Improve doc site splash/intro page

### PR DESCRIPTION
**Pull request by emanic**
## Description

- Edits intro text to be shorter and hopefully more compelling
- Modifies intro index file type to MD
- Adds get started button for call to action
- Updates Makefile to pull latest Jekyll version (old version no longer works)

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by lxpollitt** [comment]
The landing page is sufficiently important that I think Andy should be involved in reviewing the messaging.

In general for any high visibility docs change or docs restructuring I think we should be trying to get early review, before doing the implementation.  By the time it gets to PR review it should be more like "code review".  That means that the "spec and design" has already been reviewed by the key people.  

So to translate that to concrete things, I suggest have a conversation with Andy to agree what the key messages should be on this landing page as the first step.  (That's the "spec and design" review bit.)

If I've read the changes correctly then does this PR also move the top level "Releases" section to be under the "Introduction" section?  I am hesitant about that being the right thing for a couple of reasons.  Firstly, if I'm a new comer to the site and that's the second page I look at I'm probably going to be a bit overwhelmed - I won't know what most of the things are that it's talking about.  Secondly, the release details are something we want people to be reading when they are considering upgrade, so maybe hiding it under an "Introduction" section isn't the obvious place to find it.  I'm not sure what the best place is to be honest, but not convinced yet that moving it under introduction is the right thing.

**Comment by lxpollitt** [comment]
I really like the intent here to simplify and make easier to digest. It does not actually say what calico is. We need to capture the value proposition for users AND tell people what it actually is. If you see what I mean. Han me to chat perhaps if not obvious what I'm getting at. (I'm reviewing this on my phone which is never the easiest!)

**Comment by emanic** [comment]
@lxpollitt OK thanks! I will circle this by Andy and try to put something on your calendar as well to see how we can iterate on this.

File: master/introduction/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: About Calico
+---
+
+Calico provides secure networking for containers and
+other distributed, dynamic workloads.
+
+Using the Linux kernel and BGP, Calico creates and manages 
+a flat virtual network, allowing any node to route to any other. 
+In private clouds, it can run without IP encapsulation or
```
Comment (caseydavenport) [inline master/introduction/index.md]: Strictly speaking Calico can do this in some public clouds as well, so this statement feels a bit misleading.

File: master/introduction/index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: About Calico
+---
+
+Calico provides secure networking for containers and
+other distributed, dynamic workloads.
+
+Using the Linux kernel and BGP, Calico creates and manages 
+a flat virtual network, allowing any node to route to any other.
```
Comment (caseydavenport) [inline master/introduction/index.md]: do you mean workload (e.g. pod or VM) rather than node?

File: master/introduction/index.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+other distributed, dynamic workloads.
+
+Using the Linux kernel and BGP, Calico creates and manages 
+a flat virtual network, allowing any node to route to any other. 
+In private clouds, it can run without IP encapsulation or 
+network address translation for much faster speeds, easier 
+troubleshooting, and better interoperability.
+
+Using Calico's simple policy language, you can achieve
+fine-grained control over Layer 3 communications between
```
Comment (caseydavenport) [inline master/introduction/index.md]: Do we care about specifying Layer 3? Calico also offers L4 policy, for example.

File: master/introduction/index.md (hunk 2 above)
Comment (emanic) [reply master/introduction/index.md]: Thanks @caseydavenport ! I actually have some updated content I should be pushing up shortly. But yes, we are moving to "workload".

File: master/introduction/index.md (hunk 1 above)
Comment (emanic) [reply master/introduction/index.md]: This should be fixed in my update.

File: master/introduction/index.md (hunk 3 above)
Comment (emanic) [reply master/introduction/index.md]: Yes, this will be removed in next iteration. Thanks for taking a look! I'll let you know when I have the revision.

**Comment by emanic** [comment]
@caseydavenport Here is the updated intro, but could be some additional small changes still.

**Pull request closed without merging**